mailbox/event/event-cassandra/src/test/java/org/apache/james/mailbox/events/CassandraEventDeadLettersDAOTest.java

```diff
@@ -102,4 +102,7 @@ void retrieveInsertionIdsWithGroupShouldReturnStoredInsertionId() {
                 .collectList().block())
             .containsOnly(INSERTION_ID_1, INSERTION_ID_2, INSERTION_ID_3);
     }
+
+//    TODO write test for hasRecordStatement
```

Comment:
You can write todo if it helps your local development but you should avoid pushing reviews with todo comments in intermediary commits.

Reply:
Right, my bad.